airflow/www/templates/airflow/dags.html

```diff
@@ -177,25 +177,23 @@ <h2>DAGs</h2>
                 <!-- Use dag_id instead of dag.dag_id, because the DAG might not exist in the webserver's DagBag -->
                 <a href="{{ url_for('Airflow.delete', dag_id=dag.dag_id) }}"
                   onclick="return confirmDeleteDag('{{ dag.dag_id }}')">
-                   <span class="glyphicon glyphicon-remove-circle" style="color:red" aria-hidden="true" data-original-title="Delete Dag"></span>
+                   <span class="glyphicon glyphicon-remove-circle text-danger" aria-hidden="true" data-original-title="Delete Dag"></span>
                 </a>
                 </td>
             </tr>
         {% endfor %}
         </tbody>
     </table>
-    <div class="row">
-      <div class="col-sm-12" style="text-align:right;">
-        <div class="dataTables_info" id="dags_info" role="status" aria-live="polite" style="padding-top: 0px;">Showing {{num_dag_from}} to {{num_dag_to}} of {{num_of_all_dags}} entries</div>
+    <div class="text-right">
+      <div
+        class="dataTables_info mt-0"
```

Comment:
Shouldn't  this be `pt-0`?

Reply:
I looked at it now. No class is needed. I will delete classes during the next PR update

![Screenshot 2019-03-21 at 11 50 19](https://user-images.githubusercontent.com/12058428/54747600-a0d2e500-4bcf-11e9-8616-2198e6e0ab1b.png)